### ISSUE TITLE
Gaussian blurs don't match Batik

### ISSUE THREAD

**Issue by weisJ** [weisJ/jsvg#35]
Thanks! It is better now, but still rendering is not correct:

Expected (I checked Apache Batik and CEF (chrome)): see above.

Actual: 
<img width="217" alt="Screenshot 2023-04-20 at 20 29 29" src="https://user-images.githubusercontent.com/350686/233455768-082e393a-7bb9-46ac-8274-9b3b051c9d91.png">

_Originally posted by @develar in https://github.com/weisJ/jsvg/issues/33#issuecomment-1516769219_

**Comment by weisJ** [comment]
It's still not matching Batik perfectly but gaussian blurs should be more closely aligned to how the spec intends them to be implemented now.

Notably the current implementation isn't very fast for large image (which shouldn't be a problem in your case), as the fast box blurring approach for standard deviations >= 2 isn't yet implemented.

**Comment by develar** [comment]
Excellent, thanks for your work and quick fixes. It is much better now than before, but yep, not yet fully correctly.
I checked the result with commit https://github.com/weisJ/jsvg/commit/11f50d1145089df80300d39265506fedb175e85d:

after commit: <img width="541" alt="Screenshot 2023-04-21 at 07 42 41" src="https://user-images.githubusercontent.com/350686/233550615-8f619586-3ecf-4d63-8292-acf58d07a791.png">

**Comment by weisJ** [comment]
Are all of those shadows rendered using jsvg? Or only the corners?

**Comment by develar** [comment]
Please see https://github.com/JetBrains/intellij-community/blob/04f87958b90f030f0f98a694163f8b50b011dd4b/platform/platform-impl/src/com/intellij/openapi/ui/impl/ShadowPainter.kt#L82

First, we draw `bottomLeft` (corner), then fill `bottom`, then `bottomRight`.

**Comment by weisJ** [comment]
I wasn't quite able to replicate the same visual artifacts, but 4030fd8d0ec1a26c84aef505444e6b3364b89fcc fixes it visually on my end.

**Comment by develar** [comment]
It did the trick. You are awesome. Thanks a lot!

The top left corner looks strange, but I don't know whether it is a JSVG rendering bug.

<img width="253" alt="Screenshot 2023-05-01 at 15 42 17" src="https://user-images.githubusercontent.com/350686/235460105-097b6159-81a3-41e4-b6f8-fdc674c3ca77.png">

**Comment by develar** [comment]
It looks like https://github.com/weisJ/jsvg/commit/4030fd8d0ec1a26c84aef505444e6b3364b89fcc introduced some regression:
<img width="46" alt="Screenshot 2023-05-01 at 16 54 26" src="https://user-images.githubusercontent.com/350686/235471703-c5bab4f0-fea1-4cff-b480-4ddbaf53d35e.png">

```

**Comment by weisJ** [comment]
> It did the trick. You are awesome. Thanks a lot!
> 
> The top left corner looks strange, but I don't know whether it is a JSVG rendering bug.
> 
> <img alt="Screenshot 2023-05-01 at 15 42 17" width="253" src="https://user-images.githubusercontent.com/350686/235460105-097b6159-81a3-41e4-b6f8-fdc674c3ca77.png">

If it is JSVG related then it may be something platform specific. Rendering all shadows myself I get the following result:
![image](https://user-images.githubusercontent.com/31143295/235487925-708cbe1d-1843-4f48-be55-60d38ebd2831.png)

As for the rounded shadow I just pushed a commit that should fix it. However I am not sure it even worked correctly before as the issue was related to an incorrect implementation of `blendMode='normal'`.

**Comment by develar** [comment]
> that should fix it

Yep, thanks for a quick fix. All that is black magic to me, and it would take a lot of time to figure out the correct solution myself :)

Looks ok:
<img width="185" alt="Screenshot 2023-05-01 at 20 48 37" src="https://user-images.githubusercontent.com/350686/235509870-0af80b50-ab71-4fe9-a060-0ecd62f2ef8f.png">
